src/Protocol/OStatus.php

```diff
@@ -1945,7 +1945,11 @@ private static function entryContent(DOMDocument $doc, $entry, array $item, arra
 		}
 
 		XML::addElement($doc, $entry, "id", $item["uri"]);
-		XML::addElement($doc, $entry, "title", $title);
+		if ($feed_mode) {
+			XML::addElement($doc, $entry, "title", html_entity_decode($title, ENT_QUOTES, 'UTF-8'));
```

Comment:
I would guess this is the correct form both for feed_mode and non feed_mode. But unsure how to test non feed_mode so left that one untouched.

Reply:
Done https://github.com/friendica/friendica/pull/6145/commits/dd65c5d150d0732b8b8c7a773665e73946d37a3a